cloudbuild/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This Dockerfile builds the image used in Cloud Build CI to run 'make test'.
+# To push a new image, run 'gcloud builds submit --project=config-validator --tag gcr.io/config-validator/make .'
```

Comment:
Can we add a `make` target for building this?

Reply:
Sure. I'll do that in a follow-up PR.